src/policy/ephemeral_policy.cpp

```diff
@@ -83,9 +83,9 @@ bool CheckEphemeralSpends(const Package& package, CFeeRate dust_relay_rate, cons
         }
 
         if (!unspent_parent_dust.empty()) {
-            out_child_txid = tx->GetHash();
+            out_child_wtxid = tx->GetWitnessHash();
             out_child_state.Invalid(TxValidationResult::TX_MEMPOOL_POLICY, "missing-ephemeral-spends",
-                                strprintf("tx %s did not spend parent's ephemeral dust", out_child_txid.ToString()));
+                                strprintf("tx %s did not spend parent's ephemeral dust", out_child_wtxid.ToString()));
```

Comment:
callers typically use txid, so it's best to include both

```suggestion
                                strprintf("tx %s  (wtxid=%s) did not spend parent's ephemeral dust", out_child_txid.ToString(), out_child_wtxid.ToString()));
```

Reply:
Makes sense, thanks